src/content/posts/weekly-2022-06-30.adoc

```diff
@@ -0,0 +1,74 @@
+
+Hello and welcome to JBoss Editorial June 30th edition! We are nearing the end of the second week of summer, and most of us are already looking forward to our vacation plans.
+And while for some the next 2 months will be a time to relax, slow down, and take things a little easy, progress and innovation never really stop!
+So for those of you interested in the latest and the greatest that your favorite project have to offer, here are this week's highlights from around the JBoss community.
+
+== Releases, releases, releases!
+
+Here are the releases from the JBoss Community for this edition:
+
+[square]
+* link:https://camel.apache.org/blog/2022/06/camel-quarkus-release-2.10.0/[Camel Quarkus 2.10.0]
+
+* link:https://debezium.io/blog/2022/06/21/debezium-1-9-4-final-released/[Debezium 1.9.4.Final]
+
+* link:https://www.keycloak.org/2022/06/keycloak-1802-released[Keycloak 18.0.2]
+
+* link:https://blog.kie.org/2022/06/kogito-1-23-0-released.html[Kogito 1.23.0]
+
+* link:https://github.com/smallrye/smallrye-mutiny/releases/tag/1.6.0[Mutiny 1.6.0]
+
+* link:https://quarkus.io/blog/quarkus-2-10-0-final-released/[Quarkus 2.10.0]
+
+
+== Articles & Blogs
+
+=== How to convert a web application to Software-as-a-Service
+
+link:https://developers.redhat.com/articles/2022/06/16/how-convert-web-application-software-service#[How to convert a web application to Software-as-a-Service] by Bob Reselman
+
+Bob Reselman delves into the "brownfield" approach to redeveloping your web application into a SaaS platform.
+Starting off with a well-laid out example business scenario, Bob walks you through the key steps of the process from analyzing your business logic patterns, separating configuration from code, picking an appropriate service architecture, to leveraging the benefits offered by containerization and Kubernetes to ensure that you'll always be able to scale your services to match the growth of your business.
+
+=== Multi-cloud storage strategies for SaaS applications
+
+link:https://developers.redhat.com/articles/2022/06/23/multi-cloud-storage-strategies-saas-applications#[Multi-cloud storage strategies for SaaS applications] by Michael Hrivnak
+
+In the fourth entry to the ongoing link:https://developers.redhat.com/articles/2022/05/18/saas-architecture-checklist-kubernetes[_SaaS architecture checklist_] blog series Michael Hrivnak compares software-defined storage (SDS) technologies that help developers create optimized data storage solutions for use in multi-tenant cloud environments.
+The first part of Michael's article deals with the broader technical consideration of how SDS can help developers minimize platform-specific development work, followed by an overview of Red Hat's current could storage technology offerings. In the latter section, Michael provides some insight into combining these offerings to create solutions that provide customers with self-scaling, self-managed, low-latency data storage capabilities.
```

Comment:
could -> cloud

Reply:
Thank you @LightGuard, good catch, my brain and my spellchecker both missed this one:) Fixed!